src/main/js/components/CourseTable.jsx

```diff
@@ -61,35 +77,40 @@ CourseTable.HeaderRow = ({ schedule }) => (
   </tr>
 );
 
-CourseTable.courseRows = (course, index, color = '') => (
+CourseTable.courseRows = (course, index, color = '', sort, grade) => (
   course.meetings.map((meeting, meetingNum) => (
     <tr style={{ backgroundColor: color }} key={`${course.crn}${meetingNum}${index}`}>
       <td>
-        { meetingNum === 0 && course.crn }
+        {meetingNum === 0 && course.crn}
       </td>
       <td>
-        { meetingNum === 0 && `${course.subject} ${course.courseNumber}`}
+        {meetingNum === 0 && `${course.subject} ${course.courseNumber}`}
       </td>
       <td>
-        { meetingNum === 0 && course.name }
+        {meetingNum === 0 && course.name}
       </td>
+      {sort && (
+        <th>
```

Comment:
Any reason why this is <th>?

Reply:
Not sure, but fixed.

Reply:
Think it was a typo on my part, apologizes.